Postman/Nfield Public API.postman_collection.json

```diff
@@ -0,0 +1,12332 @@
+															"pm.test(\"Status code name has string\", function () {\r",
+															"    pm.response.to.have.status(\"OK\");\r",
+															"});"
+														],
+														"type": "text/javascript"
+													}
+												}
+											],
+											"protocolProfileBehavior": {
+												"disableBodyPruning": true
+											},
+											"request": {
+												"method": "GET",
+												"header": [
+													{
+														"key": "Authorization",
+														"value": "Basic {{AuthenticationToken}}",
+														"type": "text"
+													}
+												],
+												"body": {
+													"mode": "raw",
+													"raw": "",
+													"options": {
+														"raw": {
+															"language": "json"
+														}
+													}
+												},
+												"url": {
+													"raw": "{{origin}}/v1//Interviewers/{{InterviewerId}}/Offices",
+													"host": [
+														"{{origin}}"
+													],
+													"path": [
+														"v1",
+														"",
+														"Interviewers",
+														"{{InterviewerId}}",
+														"Offices"
+													]
+												},
+												"description": "Returns fieldwork offices assigned to the specified interviewer."
+											},
+											"response": []
+										},
+										{
+											"name": "Assign Office (P)",
+											"event": [
+												{
+													"listen": "test",
+													"script": {
+														"exec": [
+															"pm.test(\"Status code is 201\", function () {\r",
+															"    pm.response.to.have.status(201);\r",
+															"});\r",
+															"\r",
+															"pm.test(\"Status code name has string\", function () {\r",
+															"    pm.response.to.have.status(\"Created\");\r",
+															"});\r",
+															"\r",
+															""
+														],
+														"type": "text/javascript"
+													}
+												}
+											],
+											"request": {
+												"method": "POST",
+												"header": [
+													{
+														"key": "Authorization",
+														"value": "Basic {{AuthenticationToken}}",
+														"type": "text"
+													}
+												],
+												"body": {
+													"mode": "raw",
+													"raw": "{\r\n  \"OfficeId\": \"{{OfficeId}}\"\r\n}",
+													"options": {
+														"raw": {
+															"language": "json"
+														}
+													}
+												},
+												"url": {
+													"raw": "{{origin}}/v1//Interviewers/{{InterviewerId}}/Offices",
+													"host": [
+														"{{origin}}"
+													],
+													"path": [
+														"v1",
+														"",
+														"Interviewers",
+														"{{InterviewerId}}",
+														"Offices"
+													]
+												},
+												"description": "Add fieldwork office to an interviewer. If fieldwork office already exist on interviewer it will be ignored."
```

Comment:
```suggestion
												"description": "Add fieldwork office to an interviewer. If fieldwork office already exists on interviewer it will be ignored."
```